### ISSUE TITLE
[BUG] - Parte textual e filtro de Abrigo Disponível, indisponível e sem info.

### ISSUE THREAD

**Issue by HbLuca** [SOS-RS/frontend#146]
Descrição issue:

Ao acessar[ sos-rs.com](url) , e realizar a busca por filtros, no menu "Status do abrigo" selecionando indisponível aparecem também os que estão sem informação ou com "Consultar Disponibilidade".

Resultado esperado:

Ao selecionar em Filtros - Status do Abrigo - Abrigo Indisponível aparecerem apenas os que estão indisponíveis, da mesma forma que ao seleciona-lo ser modificado no "cabeçalho" a quantidade de abrigos indisponíveis e não com a escrita, disponíveis. 

Da mesma forma que ao selecionar os que não tem informação de disponibilidade aparecerem como Abrigos sem informação e não como disponíveis no cabeçalho.

https://github.com/SOS-RS/frontend/assets/108205892/e1df7acc-80fc-46a4-a3bd-4ddcbcfee802

Teste:

Acesse o site [sos-rs.com](url)
Selecione a opção "Filtros"
Selecione Status do Abrigo Indisponível, clique no botão Filtrar resultados.
Confira o cabeçalho com a quantidade de abrigos e também o descritivo dos abrigos se está apenas como indisponível ou com Consultar disponibilidade.

**Comment by fagundesjg** [comment]
Mergeado com dev o fix e em breve estará em produção

**Comment by HbLuca** [comment]
@kevinpiske 
[Loom](https://www.loom.com/share/096e20c853ad4314a809bc6cbe6663ba?sid=b2fd7a1d-378f-4d6b-80da-743d3649d038) referente ao teste de filtro clicando em Status de abrigo, mas fica uma dúvida referente ao que foi realizado no Frontend...

Ao selecionar em filtros "Abrigo Indisponível" e "Sem Informação de disponibilidade", o resultado está de acordo referente aos abrigos apresentados, mas o cabeçalho "Abrigos disponíveis(qtde em parenteses)" será modificado? Realizei o teste e não foi modificado, talvez tenha de verificar com o pessoal de Design/UX, PO, para tal?

Caso siga assim, por padrão... Encerro.
Caso contrário, deixarei aberto ainda.


Obs: Comentei sobre no https://github.com/SOS-RS/frontend/issues/178 com o @JuanGustah

**Comment by kevinpiske** [comment]
@HbLuca sim, ficou faltando o ajuste do label. Acredito que hoje a noite eu corrija ele. Outro ponto é que minha alteração inicial foi modificada, gerando um novo problema. Os abrigos com capacidade 0 ficarão sem status e sumirão da listagem. Acredito que esse comportamento esteja errado e que o correto seria tratá-los como abrigos sem informação.

**Comment by HbLuca** [comment]
@kevinpiske muito obrigado!

Concordo contigo, se estão cadastrados, devem aparecer como sem informação a não ser que venham com "desativado" [desativado] ou qualquer outra forma possivel de desativado.
Comentei sobre, no https://github.com/SOS-RS/frontend/issues/178 com o @JuanGustah
O mesmo estava verificando parte de back e front sobre.

**Comment by kevinpiske** [comment]
Minha proposta de alteração textual é a seguinte:

![image](https://github.com/SOS-RS/frontend/assets/14002944/beef1255-fd5e-49e7-8fbf-fef9b5c444d9)

Substituição de "disponíveis" por "encontrados", assim não gera confusão com o status do abrigo e também não fica ruim mesmo selecionando múltiplos status no filtro.

Também a remoção da palavra disponíveis no alerta.

**Comment by HbLuca** [comment]
@kevinpiske para mim, faz todo sentido.
Tem de verificar com equipe de Design/UX, pois isso impacta no projeto do Figma também.

Além de impactar possívelmente as "subidas" de geral, referente a correções, sugestões, melhorias...
